frontend/static/js/select-2.js

```diff
@@ -0,0 +1,55 @@
+Vue.component('select-2', {
+  template: '<select v-bind:name="name" class="form-control" v-bind:multiple="multiple"></select>',
```

Comment:
instead of keeping the template in the js, can we please move it to the HTML file as we've done in the main UI?

Reply:
If we put it in HTML, so when we want to use the component in other HTML page we need to rewrite it (`<select v-bind:name="name". . .`) again. How?